aten/src/ATen/native/quantized/cpu/qmul.cpp

```diff
@@ -173,27 +173,48 @@ static auto registry =
             "-> Tensor out",
             c10::RegisterOperators::options()
                 .kernel<QMulOut</*ReLUFused=*/true>>(
-                    TensorTypeId::QuantizedCPUTensorId))
-
-        .op("quantized::mul_scalar(Tensor qa, Scalar b)"
-            "-> Tensor qc",
-            c10::RegisterOperators::options()
-                .kernel<QMulScalar</*ReLUFused=*/false>>(
-                    TensorTypeId::QuantizedCPUTensorId))
-        .op("quantized::mul_scalar_relu(Tensor qa, Scalar b)"
-            "-> Tensor qc",
-            c10::RegisterOperators::options()
-                .kernel<QMulScalar</*ReLUFused=*/true>>(
-                    TensorTypeId::QuantizedCPUTensorId))
-        .op("quantized::mul_scalar_out(Tensor qa, Scalar b, Tensor out)"
-            "-> Tensor out",
-            c10::RegisterOperators::options()
-                .kernel<QMulScalarOut</*ReLUFused=*/false>>(
-                    TensorTypeId::QuantizedCPUTensorId))
-        .op("quantized::mul_scalar_relu_out(Tensor qa, Scalar b, Tensor out)"
-            "-> Tensor out",
-            c10::RegisterOperators::options()
-                .kernel<QMulScalarOut</*ReLUFused=*/true>>(
                     TensorTypeId::QuantizedCPUTensorId));
 }  // namespace
+
+constexpr const char *kTensorTensorError = "torch.mul is not supported when multiplying two"
+" quantized tensors. Please use torch.nn.quantized.modules.FloatFunctional";
```

Comment:
I don't think we should be giving suggestions, in case `FloatFunctional` is replaced/renamed.